_Group Projects/Group_5/program.do

```diff
@@ -0,0 +1,26 @@
+// Set global dir
+
+global wd "/Users/miglepetrauskaite/Desktop/GITHUB/Repo/econ490-fall22/_Group Projects/Group_5"
+
+// set trace on
+// Create a program
+
+cap prog drop group5 
+
+/// regressing treatment and dependent variable, exporting regression table
+program define group5
+	ssc install outreg2
+	webuse nlswork, clear
+	syntax anything
+	local var1: word 1 of `anything'
+	local var2: word 2 of `anything'
+	reg `var1' `var2'
+	outreg2 using "$wd/demonstration.doc", replace
+end
```

Comment:
Would be helpful to have slightly more comments explaining what you're doing in each line of code.

Reply:
The README file explains what the program does -- it's a simple regression that exports its output into a doc file. However, I have also added a more thorough explanation to the program.do file.